### PR TITLE
Agusramirez/gemimi/nl2slq

### PR DESCRIPTION
**Pull request by imp14a**

### CODE REVIEW

**Comment by awaemmanuel** [comment]
LGTM

**Comment by awaemmanuel** [comment]
@imp14a is this tested?

**Comment by michaelwsherman** [comment]
@imp14a can you also remove the declaration of llm_ga? text2sql is the only method that uses it, I believe.

**Comment by michaelwsherman** [comment]
Remove the two references to palm in audiences.components.html

**Comment by imp14a** [comment]
Merge dev, fix conflicts, and simplify queries

This commit merges the latest changes from dev, resolves minor conflicts, and includes a simpler query

- Simpler query for audiences page.
- Tested. 
- Addressed all previous comments and suggestions.

File: frontend/src/app/audiences/audiences.component.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
     Audiences
 </div>
 
-<div class="subTitle">Extract data from BigQuery using natural language and the PaLM API.
+<div class="subTitle">Extract data from BigQuery using natural language and the Gemini.
```
Comment (michaelwsherman) [inline frontend/src/app/audiences/audiences.component.html:5]: nit: remove "the" before Gemini

Comment (imp14a) [reply frontend/src/app/audiences/audiences.component.html]: Done, please review

File: backend_apis/app/main.py [hunk 2]
```diff
@@ -107,7 +106,8 @@
 datacatalog_client = datacatalog_v1.DataCatalogClient()
 
 # Text models
-llm_ga = bison_ga.from_pretrained(model_name="text-bison@002")
+gemini_llm = GenerativeModel("gemini-1.0-pro-001")
```
Comment (michaelwsherman) [inline backend_apis/app/main.py:109]: What is the purpose of this gemini_llm declaration? It appears unused, can you remove it?

Comment (imp14a) [reply backend_apis/app/main.py]: Done!

**Pull request merged**